src/pages/guides/using/aem/index.md

```diff
@@ -1,24 +1,25 @@
 ---
-title: AEM Events
+title: Setting up AEM Events with Adobe IO Events
 ---
 
-# AEM Events
+# Setting up AEM Events with Adobe IO Events
 
-**Adobe I/O Events integration package for AEM is now open source:
-see [`aem-io-events`](https://github.com/adobe/aio-lib-java/tree/main/aem/aio_aem_events) module
-in our [`aio-lib-java`](https://github.com/adobe/aio-lib-java) github project**
+These instructions describe how to set up Adobe Experience Manager (AEM) to integrate with Adobe IO Events. Adobe IO Events integration package for AEM is now open source:
+See [`aem-io-events`](https://github.com/adobe/aio-lib-java/tree/main/aem/aio_aem_events) module in our [`aio-lib-java`](https://github.com/adobe/aio-lib-java) GitHub project.
 
 ## Requirements
 To set it up, you need:
 
-* An AEM (version 6.5 and above) instance with administrative permissions. 
+* An AEM (version 6.5 and above) instance with administrative permissions.
   * if you are still running AEM version 6.4, and wish to use this solution, [let us know](https://github.com/adobe/aio-lib-java/issues/104)
 * [Adobe Developer Console](/console) access, with administrative permissions for your enterprise organization.
 
 ## Installation guides
 These installation guides describe how to deploy and configure it so you can use Adobe I/O Events
 for notification of AEM events, such as page or asset CUD operations.
-* [AEM on premise installation guide](../aem/aem_on_premise_install.md)
+* For AEM On Premise, see:
+  * [AEM (6.5.x and above) on premise installation guide](../aem/aem_on_premise_install.md)
+  * [AEM (6.4.x) on premise installation guide](../aem/deprecated/aem_on_premise_install_6.4.md) [`Deprecated`]
```

Comment:
I would move this line (line 22) in the requirements section may be like this:

```
* if you are still running AEM version 6.4, 
    * and wish to use this solution, [let us know](https://github.com/adobe/aio-lib-java/issues/104)
    * you may also our [old deprecated and unsupported aem 6.4 package](../aem/deprecated/aem_on_premise_install_6.4.md)
```
and delete the section here